okhttp/api/okhttp.api

```diff
@@ -1048,6 +1049,7 @@ public class okhttp3/Request$Builder {
 	public fun addHeader (Ljava/lang/String;Ljava/lang/String;)Lokhttp3/Request$Builder;
 	public fun build ()Lokhttp3/Request;
 	public fun cacheControl (Lokhttp3/CacheControl;)Lokhttp3/Request$Builder;
+	public final fun cacheUrlOverride (Lokhttp3/HttpUrl;)Lokhttp3/Request$Builder;
```

Comment:
Note that we’re not adding `OkHttpExperimentalApi` on this, and so we’re committing to this exact API and name. I’m okay with that.

Reply:
Should we reconsider as a policy for new API?